SignalShareExtension/ShareViewController.swift

```diff
@@ -71,6 +65,12 @@ public class ShareViewController: UINavigationController, SAELoadViewDelegate, S
             return
         }
 
+        AppSetup.setupEnvironment({
```

Comment:
The fix: We shouldn't set up our environment until _after_ we've consulted `isReadyForAppExtensions`.

Reply:
Good point! Can you leave a comment to that affect?

Reply:
Done.